### DISCUSSION TITLE
请问pr中的pending merge是啥意思？考虑合并？是还需要继续商议吗

### DISCUSSION THREAD

**Discussion by Sunny-117** [reactjs/zh-hans.react.dev#1055] [Q&A] [answered]
请问pr中的pending merge是啥意思？考虑合并？是还需要继续商议吗

**Comment by awxiaoxian2020** [comment, marked as answer]
目前由于 
![pr](https://user-images.githubusercontent.com/69554233/218768158-3f30e47f-6123-49be-a88e-598413c9ead3.png)
netlify ci 的原因，我没有合并的权限，故加上这样一个标签，表示可以合并，等待这个问题处理好以后，将其合并，方便我们对 PR 进行整理。
cc: @QC-L
